baseplate/frameworks/pyramid/__init__.py

```diff
@@ -68,11 +75,60 @@ def close(self) -> None:
             self.app_iter.close()  # type: ignore
 
 
+PROM_NAMESPACE = "http_server"
+
+HISTOGRAM_LABELS = [
+    "http_method",
+    "http_endpoint",
+    "http_success",
+]
+REQUEST_LATENCY = Histogram(
+    f"{PROM_NAMESPACE}_latency_seconds",
+    "Time spent processing requests",
+    HISTOGRAM_LABELS,
+    buckets=default_latency_buckets,
+)
+REQUEST_SIZE = Histogram(
```

Comment:
Is there a usecase for these request/response size metrics? not against them (not in the spec) but want to be sure they'll actually be useful for folks

Reply:
I'm fine with it as long as @bjk-reddit is. I don't think it's worth adding to the spec at the moment though. Happy to disagree here. I'm not against it since I've used this data from LB logs in the past and this makes it a bit easier